plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -398,9 +398,20 @@ function WordInfoDialog:init()
         end
     }
 
+    local buttons = {{reset_button, remove_button}}
+    if self.show_parent.item.last_due_time then
+        table.insert(buttons, {{
+            text = _("Undo study"),
```

Comment:
I'm not entirely happy with that. Would a plain undo be unclear in context?

Reply:
I believe yes. The undo button is in the `more` popup widget and each word has its own. Between one pair of `do` and `undo`, a user could have done many other things which could make a plain `undo`'s meaning less clear.

Reply:
Undo study status?

Reply:
Done.